src/store/ducks/generic/MDASchoolReport.ts

```diff
@@ -0,0 +1,177 @@
+import intersect from 'fast_array_intersect';
```

Comment:
I think we shouldn't mention "school" in this reducer module.  It really is not about "schools", it is about OpenSRP locations.  It just so happens that for MDA in Eswatini, the locations are schools.  So instead of calling this MDASchoolReport, I would call it MDALocationReport.